packages/core/src/store/__tests__/state-store-integration.test.ts

```diff
@@ -16,6 +16,7 @@ import { CID } from 'multiformats/cid'
 import { createIPFS } from '@ceramicnetwork/ipfs-daemon'
 import { createCeramic } from '../../__tests__/create-ceramic.js'
 import { RunningState } from '../../state-management/running-state.js'
+import { Repository } from '../../state-management/repository'
```

Comment:
```suggestion
import { Repository } from '../../state-management/repository.js'
```

Reply:
ugh good catch.  I wonder why the docker build CI step didn't catch this this time.

I also wonder if there's a way I can configure Webstorm to always add the `.js` by default...